stories/Drag.stories.tsx

```diff
@@ -465,6 +465,7 @@ export const NestedNodeRearranging = () => {
         nodes={nodes}
         edges={edges}
         node={<Node dragType="node" />}
+        dragNode={(node: NodeProps) => (<Node {...node} dragType="node" />)}
```

Comment:
Why do you need this callback?

Reply:
you don't. it was just a test to make sure that having a callback for `dragNode` works

Reply:
OK - lets make these demos as simple as possible - let's just use the nromal one